src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -219,6 +219,27 @@ public override async Task<IList<DurableOrchestrationStatus>> GetStatusAsync(Dat
             return results;
         }
 
+        // TODO implementation to enable paging
+        public override async Task<DurableOrchestrationStatusContext> GetStatusAsync(DateTime createdTimeFrom, DateTime? createdTimeTo, IEnumerable<OrchestrationRuntimeStatus> runtimeStatus, int pageSize, string continuationToken, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
@cgillum The reason for adding `DurableOrchestrationStatusContext` is that we did not want to use Tuple to keep future extensibility. What do you think?

Reply:
I agree - we should avoid Tuple.